src/form_builder/models.py

```diff
@@ -51,10 +51,12 @@ class Form(models.Model):
                              max_length=255,
                              help_text=_("Give your form a name."))
     slug = models.SlugField(_("Slug"),
```

Comment:
I'm pretty sure the `label` property comes from the text within the parens here. It looks like the same thing happens with the instructions data. I think if you change this to `slug = models.SlugField(_("URL"),` I'm pretty sure it'll output as `URL` in the label.